src/main/java/org/folio/fqm/service/EntityTypeFlatteningService.java

```diff
@@ -0,0 +1,242 @@
+package org.folio.fqm.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import kotlin.Pair;
```

Comment:
We should probably use a different implementation of `Pair`. We're not actually using Kotlin (unfortunately), so it just feels weird to use stuff from its standard lib